staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -20,6 +20,9 @@ import (
 	"fmt"
 
 	"github.com/spf13/pflag"
+	"net/http"
+	"k8s.io/apiserver/pkg/server/healthz"
+	"k8s.io/kubernetes/cmd/kube-apiserver/app/preflight"
```

Comment:
This dependency won't be allowed.  You'll need to move the etcd check code to the `k8s.io/apiserver` staging section somewhere.